workflow-bot-app/src/main/java/com/symphony/bdk/workflow/DatabaseConfiguration.java

```diff
@@ -0,0 +1,14 @@
+package com.symphony.bdk.workflow;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+@Configuration
+@EnableTransactionManagement
```

Comment:
as the same is added in WorkflowBotApplication, we can remove this class

Reply:
I forgot to remove this class, it is useless actually